scripts/validation/validate.js

```diff
@@ -143,3 +154,11 @@ function assertValid(schema, subject) {
     throw new Error(errorMessage);
   }
 }
+
+async function getModifiedServices() {
+  const git = simpleGit(path.resolve(__dirname, '../..'), { maxConcurrentProcesses: 1 });
+  const modifiedFiles = await (await git.diff([ '--name-only', 'master...' ])).split('\n');
+  const modifiedServicePaths = modifiedFiles.filter(filepath => filepath.startsWith('services/') && !filepath.includes('.filters.js'));
```

Comment:
Why not simply rely on the `path` argument to `git diff`? 🙂
Above, add the following parameters so that the diff command is `git diff --name-only master services/*.json`.